float32 outer_angle
float32 inner_angle
float32 thrust

